script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -745,28 +806,29 @@ There was still nothing 'eternal' to be found on Earth.
 [sha:165f874d1c1ab581b56211951d835dcca5b76e84]{
 -- Page 693, Offset 16534.
 -- 増え続け、変化し続けるのなら“完成”を待つしかないと。
-So if everything kept expanding and changing, then he simply had to wait for it to 'complete'.
+So if everything kept expanding and changing, then he simply had to wait for it to be 'finished'.
 }
 [sha:1fb58b303fd77448ef7eb89283765dd0f693828f]{
 -- Page 694, Offset 16535.
 -- 彼は階段を踏み外した。
 // using both stairway and ladder is weird in English, and in this case I think stairway is metaphorical while ladder is a Bible reference
-He went astray on his way up,
+He strayed from the path before him.
```

Comment:
I'd rather keep the 'upward' motion intact as to make him rejecting heaven more explicit.